content/eu/main.md

```diff
@@ -12,8 +16,48 @@ subtitle: The homepage of the European Galaxy community
 
 The **Galaxy Community** is actively involved in helping the ecosystem improve and sharing scientific discoveries.
 
+<br><br><br>
+
 <div class="row justify-content-center">
-  <a href="/get-started/" class="btn w-75 btn-primary btn-lg">
-    Get Started: First Steps with Galaxy
+  <a href="/get-started/" class="btn w-25 btn-primary btn-lg" style="margin:1px">
+    Installed tools
+  </a>
+
+  <a href="./data-policy.md" class="btn w-25 btn-primary btn-lg" style="margin:1px">
+    TIaaS
   </a>
-</div>
+
+  <a href="./data-policy.md" class="btn w-25 btn-primary btn-lg" style="margin:1px">
+    Data retention police
+  </a>
+</div> 
+
+<br><br>
+
+
+<!-- Events, News, Tweets here -->
```

Comment:
Events, News are now at the bottom of the page. Is there a way to move them up?

Reply:
FYI, I actually already added the data policy - you can see it at the bottom [this page](https://galaxyproject.org/bare/usegalaxy-eu/main/#our-data-policy). It's currently just an Insert, so you can't link directly to it, though. I can put on its own page, or insert it into the homepage if you'd like.

Also, I just added another insert slot below the news/events, so if you want stuff to appear below it, just put it in `extra.md`.